ui/js/controllers/bugfiler.js

```diff
@@ -96,6 +101,11 @@ treeherder.controller('BugFilerCtrl', [
                 }
             }
 
+            // We don't want to include "REFTEST" when it's an unexpected pass
+            if (summary[0].search("REFTEST TEST-UNEXPECTED-PASS") >= 0) {
```

Comment:
I've been planning on a refactor for a while. Guess I'll finally file that now and work on that.